pom.xml

```diff
@@ -575,10 +575,6 @@
                     </exclusion>
                     <exclusion>
                         <groupId>org.apache.geronimo.specs</groupId>
-                        <artifactId>geronimo-jta_1.0.1B_spec</artifactId>
```

Comment:
If you remove this exclusion it would cause the library to be potentially pulled again by the activemq dependency